Unable to start 0.49.8+ on linux

I am unable to start any version since 0.49.8 ( when switch from "lab" to "mml" happened )

```
Error: Unable to initialize main class megameklab.MegaMekLab
Caused by: java.lang.NoClassDefFoundError: megamek/client/ui/preferences/SuitePreferences

```
no logfile is produced

Debian Buster 
```
 java --version
openjdk 11.0.16 2022-07-19
OpenJDK Runtime Environment (build 11.0.16+8-post-Debian-1deb10u1)
OpenJDK 64-Bit Server VM (build 11.0.16+8-post-Debian-1deb10u1, mixed mode, sharing)

```

0.49.7 and prior works

I'm hitting the same issue with Ubuntu 20.04.

./mml-startup
Error: Unable to initialize main class megameklab.MegaMekLab
Caused by: java.lang.NoClassDefFoundError: megamek/client/ui/preferences/SuitePreferences

java -jar MegaMekLab.jar 
Error: Unable to initialize main class megameklab.MegaMekLab
Caused by: java.lang.NoClassDefFoundError: megamek/client/ui/preferences/SuitePreferences

java --version
openjdk 11.0.17 2022-10-18
OpenJDK Runtime Environment (build 11.0.17+8-post-Ubuntu-1ubuntu220.04)
OpenJDK 64-Bit Server VM (build 11.0.17+8-post-Ubuntu-1ubuntu220.04, mixed mode, sharing)

--- a/megameklab/build.gradle
+++ b/megameklab/build.gradle
@@ -86,7 +86,7 @@ task mmJar (type: Jar, dependsOn: compileMM) {
     manifest {
         attributes "Main-Class" : 'megamek.MegaMek'
         attributes "Class-Path" : project.sourceSets.main.runtimeClasspath.files
-                .findAll { it.name.endsWith(".jar") && !it.name.toLowerCase().startsWith("megamek") }
+                .findAll { it.name.endsWith(".jar") }
                 .collect { "${lib}/${it.name}" }.join(' ')
         attributes "Add-Opens" : 'java.base/java.util java.base/java.util.concurrent'
         attributes "Build-Date" : LocalDateTime.now()
@@ -102,7 +102,7 @@ jar {
     manifest {
         attributes "Main-Class" : mainClassName
         attributes "Class-Path" : "${mmJar.archiveFileName.get()} " + (project.sourceSets.main.runtimeClasspath.files
-                .findAll { it.name.endsWith(".jar") && !it.name.toLowerCase().startsWith("megamek") }
+                .findAll { it.name.endsWith(".jar") }
                 .collect { "${lib}/${it.name}" }.join(' '))
         attributes "Add-Opens" : 'java.base/java.util java.base/java.util.concurrent'
         attributes "Build-Date" : LocalDateTime.now()

